test/parallel/test-cluster-worker-events.js

```diff
@@ -49,31 +49,31 @@ assert(cluster.isWorker);
 let sawProcess;
 let sawWorker;
 
-process.on('message', function(m) {
-  assert(!sawProcess);
-  sawProcess = true;
-  check(m);
-});
-
-cluster.worker.on('message', function(m) {
-  assert(!sawWorker);
-  sawWorker = true;
-  check(m);
-});
-
 const messages = [];
 
-function check(m) {
+const check = (m) => {
```

Comment:
Just a nit: for top level functions like this, using the `function check(m) {` form is preferred. The arrow functions on the event handlers is fine.